src/server/csp.ts

```diff
@@ -67,12 +67,13 @@ const assetHashes = (assets: string[]): string =>
 
 const buildCsp = ({ styles, scripts }: Assets, twitter: boolean): string => `
     default-src 'self';
-    style-src 'self' ${assetHashes(styles)} https://interactive.guim.co.uk ${twitter ? 'https://platform.twitter.com' : ''};
+    style-src 'self' 'unsafe-inline';
```

Comment:
I see you've flagged this in the description already, but just a reminder that we probably want to change this before merging.